src/app/server/serverInit.ts

```diff
@@ -110,6 +111,35 @@ export async function serverInit() {
             }
 
             InitSchemas(pool);
+            RegisterDonorComponent.sendMail = async (subject: string, message: string, email: string) => {
```

Comment:
maybe this function should be exposed outside of a component. it does not seem to use any of the component infra and it is also called by register-helper component.
suggest to move it to general utils.